task/github-add-gist/0.1/README.md

```diff
@@ -0,0 +1,70 @@
+# GitHub Add Gist
+
+The following task lets you upload a file to `gist` GitHub
+and outputs the raw url as the result.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/github-add-gist/0.1/github-add-gist.yaml
+```
+
+## Secrets
+
+This Task requires access to a GitHub token set via a Kubernetes Secret. By default, the name of this Secret should be `github` and the secret key should be `token`, but you can configure this via the `GITHUB_TOKEN_SECRET_NAME` and `GITHUB_TOKEN_SECRET_KEY` [parameters](#parameters) described below.
+
+To create such a Secret via `kubectl`:
+
+```
+kubectl create secret generic github --from-literal token="MY_TOKEN"
+```
+
+Check [this](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) to get personal access token for `Github`.
+
+See GitHub's documentation on [Understanding scopes for OAuth Apps](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/) to figure out what scopes you need to give to this token to add comment to an issue or a pull request.
+
+> Note: Make sure you have added `gist` scope while creating a personal access token.
+
+## Parameters
+
+- **GITHUB_HOST_URL:**: The GitHub host domain (_default:_ `api.github.com`)
```

Comment:
I am curious about the use of ALL_CAPS for parameter names.  Is there a specific reason to do it this way?  It looks like the catalog doesn't have any consistent pattern for this so I guess it doesn't matter too much but I was wondering if there's a specific benefit in mind when using them?